Wikipedia/Code/NSUserDefaults+WMFApplicationDefaults.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+extension NSUserDefaults {
+    public func wmf_setReadingFontSize(fontSize: NSNumber) {
+        self.setObject(fontSize, forKey: "ReadingFontSize")
+        self.synchronize()
+    }
+    
+    public func wmf_readingFontSize() -> NSNumber {
```

Comment:
Can we rename this to 'wmf_fontSizeMultiplier' or 'wmf_webViewFontSizeMultiplier'?